MAUI/DataForm/Editors.md

```diff
@@ -273,7 +273,7 @@ public string Address { get; set; }
 {% endhighlight %}
 {% endtabs %}
 
-![MultilineTextEditor](images/editors/dataform-multiline-text-editor.png)
+![Multiline Text editor for .NET MAUI DataForm.](images/editors/dataform-multiline-text-editor.png)
```

Comment:
Multiline Text editor in .NET MAUI DataForm.

Update all editors caption like this

Reply:
updated